content/applications/services/project/project_management/top_bar.rst

```diff
@@ -0,0 +1,40 @@
+=======
+Top Bar
+=======
+
+In project management, reviewing the various records and documents related to a project is often
+necessary. Odoo Project’s **top bar** provides quick access to these essential resources. You can
+customize each project's top bar to match its specific needs.
+
+Configuration
+=============
+
+To set up the top bar, go to the :guilabel:`Project` app, open an existing project, and click the
+top bar :icon:`fa-sliders` :guilabel:`(sliders)` button. This makes the top bar appear above the
+search bar. Click the sliders button at the top center of the screen to select which records you
+want to display, such as timesheets, sales orders, invoices, documents, dashboards, and more. You
+can add as many options as you need to your project’s top bar.
+
+.. image:: top_bar/top-bar.png
+   :align: center
+   :alt: Top bar selection menu
+
+Creating custom views
+=====================
+
+You can also create your own buttons to access more specific views:
+
+#. Use the **search bar** with keywords, filters, and grouping options.
+#. Click the top bar button and go to Save View.
+#. Name your view and check the box if you want to share your button with other users.
```

Comment:
```suggestion
#. Edit the default button name if necessary, then enable :guilabel:`Shared` if you want to share the button with other users.
```
- Even though there’s only one checkbox right now, I think naming it is still valuable (e.g., it helps users confirm they’re in the right place).
- It might also be helpful to clarify that the "view name" will actually be the button label, as "view" feels more technical and less practical for users.